megazords/ios-rust/Cargo.toml

```diff
@@ -8,6 +8,9 @@ license = "MPL-2.0"
 [lib]
 crate-type = ["staticlib"]
 
+[features]
+builtin-glean = ["nimbus-sdk/builtin-glean"]
```

Comment:
At first I was going to suggest replacing this with a `features = ["builtin-glean"]` entry in the dependencies section for nimbus, but I realized that I actually like this - in case we ever want to test the not-glean-in-rust in iOS! Awesome! 🚀